tests/Microsoft.Graph.DotnetCore.Core.Test/TestModels/ServiceModels/TestEmailAddress.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph.DotnetCore.Core.Test.TestModels.ServiceModels
+{
+    using System.Collections.Generic;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// The type TestEmailAddress.
+    /// </summary>
+    [JsonConverter(typeof(DerivedTypeConverter<TestEmailAddress>))]
```

Comment:
EmailAddress shouldn't have a DerivedTypeConverter since it isn't derived.

Reply:
Resolved